dist/samples/places-autocomplete-addressform/inline.html

```diff
@@ -104,95 +143,133 @@
+        // entry of subpremise information such as apartment, unit, or floor number.
+        address2Field.placeholder = "Apartment, unit, or floor #";
+        address2Field.focus();
       }
     </script>
   </head>
   <body>
-    <div id="locationField">
-      <input
-        id="autocomplete"
-        placeholder="Enter your address"
-        onFocus="geolocate()"
-        type="text"
-      />
-    </div>
-
-    <!-- Note: The address components in this sample are typical. You might need to adjust them for
-               the locations relevant to your app. For more information, see
-         https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform
+    <!-- Note: The address components in this sample are based on North American address format. You might need to adjust them for the locations relevant to your app. For more information, see
+https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform
     -->
 
-    <table id="address">
-      <tr>
-        <td class="label">Street address</td>
-        <td class="slimField">
-          <input class="field" id="street_number" disabled="true" />
-        </td>
-        <td class="wideField" colspan="2">
-          <input class="field" id="route" disabled="true" />
-        </td>
-      </tr>
-      <tr>
-        <td class="label">City</td>
-        <td class="wideField" colspan="3">
-          <input class="field" id="locality" disabled="true" />
-        </td>
-      </tr>
-      <tr>
-        <td class="label">State</td>
-        <td class="slimField">
-          <input
-            class="field"
-            id="administrative_area_level_1"
-            disabled="true"
-          />
-        </td>
-        <td class="label">Zip code</td>
-        <td class="wideField">
-          <input class="field" id="postal_code" disabled="true" />
-        </td>
-      </tr>
-      <tr>
-        <td class="label">Country</td>
-        <td class="wideField" colspan="3">
-          <input class="field" id="country" disabled="true" />
-        </td>
-      </tr>
-    </table>
+    <form action="" method="get" autocomplete="off">
+      <span class="title">Sample address form for North America</span>
+      <label class="fullfield" for="gmp-a1">
+        <!-- Avoid the word "address" in id, name, or label text to avoid browser autofill from conflicting with Place Autocomplete. Star or comment bug https://crbug.com/587466 to request Chromium to honor autocomplete="off" attribute. -->
+        <span class="formlabel" id="a1-label">Deliver to*</span>
+        <input
+          type="text"
+          id="gmp-a1"
+          name="gmp-a1"
+          aria-labelledby="a1-label"
+          required
+          autocomplete="off"
+        />
+      </label>
+      <label class="fullfield" for="address2">
+        <span class="formlabel" id="address2-label"
+          >Apartment, suite, etc.</span
+        >
```

Comment:
weird formatting

Reply:
that is weird. that's the output from prettier default settings

Reply:
Seems fixed now. https://github.com/googlemaps/js-samples/blob/refactor/places-autocomplete-addressform/dist/samples/places-autocomplete-addressform/inline.html#L206